js/apps/admin-ui/src/groups/GroupAttributes.tsx

```diff
@@ -29,27 +27,24 @@ export const GroupAttributes = () => {
   });
 
   const location = useLocation();
-  const id = getLastId(location.pathname);
-  const { currentGroup, subGroups, setSubGroups } = useSubGroups();
-
-  const convertAttributes = (attr?: Record<string, any>) => {
-    return arrayToKeyValue(attr || currentGroup()?.attributes!);
-  };
+  const id = getLastId(location.pathname)!;
+  const [currentGroup, setCurrentGroup] = useState<GroupRepresentation>();
 
-  useEffect(() => {
-    form.setValue("attributes", convertAttributes());
-  }, [subGroups]);
+  useFetch(
+    () => adminClient.groups.findOne({ id }),
+    (group) => {
+      convertToFormValues(group!, form.setValue);
+      setCurrentGroup(group);
+    },
+    [],
+  );
 
   const save = async (attributeForm: AttributeForm) => {
     try {
-      const group = currentGroup();
-      const attributes = keyValueToArray(attributeForm.attributes!);
+      const group = currentGroup;
```

Comment:
If `currentGroup` no longer needs to be called, perhaps we can remove this alias and just use it directly?